index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Review code</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Hello World ! , We are Software developers</h1>
+    <p id="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
+
+    <form id="survey-form">
+      <div class="form-group">
+        <label id="name-label">Name</label>
+        <input type="text" id="name" placeholder="name here" />
+      </div>
+      <div class="form-group">
+        <label id="email-label">Email</label>
+        <input type="email" id="email" placeholder="email" required />
+      </div>
+
+      <div class="form-group">
+        <label class="number-label">Age</label>
+        <input
+          type="number"
+          id="number"
+          placeholder="age"
+          min="0"
+          max="9"
+          required
+        />
+      </div>
+
+      <div class="form-group">
+        <label class="number-label">
+          Which option best describes your current role?</label
+        >
+        <select name="dropdown" id="dropdown">
+          <option value="man">man</option>
+          <option value="girl">girl</option>
+          <option value="boy">boy</option>
+        </select>
+      </div>
+
+      <div class="form-group">
+        <label class="number-label">
+          Would you recommend freeCodeCamp to a friend?</label
+        >
```

Comment:
You have done a good job with your code 👍  but Please, try to cut down on the number of empty lines in your code. Less empty space can result in faster page loads and a better user experience. Empty lines in HTML code can be removed if they are unneeded. Additionally, it makes the code shorter and simpler to understand, which is advantageous for engineers working on the code.